npm/react/cypress.json

```diff
@@ -5,6 +5,7 @@
   "projectId": "z9dxah",
   "testFiles": "**/*spec.{js,jsx,ts,tsx}",
   "env": {
+    "reactDevtools": true,
```

Comment:
we need to set this automatically

Reply:
A couple of idea on how to do it:
- By parsing the contents of the webpack config 
- By looking up the react object in the AUT window (too late?)

Reply:
I wrote up a comment at the end, but I'd like to do this via the Plugins file -- we can automatically infer react if they use one of the React presets... But we also let users manually setup React Devtools if they are using a custom `dev-server:start` event.

Reply:
The only way is to put it inside the config, but we need to rework our plugins system firstly https://cypress-io.atlassian.net/browse/CT-225